test/LibraryManager.IntegrationTest/LibmanTestsUtility.cs

```diff
@@ -8,6 +9,164 @@ namespace Microsoft.Web.LibraryManager.IntegrationTest
 {
     public class LibmanTestsUtility
     {
+        private static HashSet<string> GetTopLevelDirectoriesAndFiles(string cwd, bool caseInsensitive)
```

Comment:
>cwd [](start = 77, length = 3)

don't use abbreviated variable names

Reply:
will change

---
In reply to: [194548281](https://github.com/aspnet/LibraryManager/pull/113#discussion_r194548281) [](ancestors = 194548281)